keyboards/tkc1800/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x5443 // "TK"
```

Comment:
I'm almost definitely nitpicking here, but by convention `0x5443` actually implies "TC" rather than "TK". "TK" would be `0x544B`.

Reply:
whoops